requirements.txt

```diff
@@ -2,6 +2,4 @@ numpy
 setuptools>=6.0
 Cython>=0.20,<0.27
```

Comment:
Can you remove this requirement that I had added temporarily a while back?  So Cython >=0.20 should work.

Reply:
@rahulgovind -- can you address this please.